mavros/fcf9cd9/fcf9cd9.bug

```diff
@@ -70,4 +70,4 @@ fault-codes:
   - PROGRAMMING:UNINITIALISED
 failure-codes:
   - SOFTWARE:BEHAVIORAL
-  - SYSTEM:MODEL-REALITY-DIVERGENCE
+  - SYSTEM:MONITORING
```

Comment:
Incorrect timestamps seems like a good fit for BEHAVING-INCORRECTLY to me. MONITORING could also fit here too?

Reply:
Same reasons as above: Both BEHAVING-INCORRECTLY and MONITORING apply. (The operator may indirectly rely on information that is derived from timestamps [e.g., the UAV's orientation]).